package.json

```diff
@@ -38,11 +38,14 @@
     "babel-jest": "^24.1.0",
     "babel-loader": "^8.0.5",
     "cloudinary": "^1.13.2",
+    "cloudinary-core": "^2.6.2",
+    "cloudinary-react": "^1.1.0",
```

Comment:
Are we still using this?

Reply:
No we are not... 
I will uninstall them